### ISSUE TITLE
dev-shell is broken

### ISSUE THREAD

**Issue by lf-** [nix-community/nixops-digitalocean#6]
https://github.com/nix-community/nixops-digitalocean/commit/bbae2e0962287d26222065407eccf19980054a33 broke it by deleting release.nix. I assume at this point it is superfluous as I see a shell.nix?

**Comment by Kiwi** [comment]
yes the documentation and code is updated here but not merged yet https://github.com/Kiwi/nixops-digitalocean/tree/no-kitchen-sink
